docs/modules/operation/pages/events/introduction.adoc

```diff
@@ -0,0 +1,27 @@
+
+[[events]]
+= Events
+
+Events are central to the operation of the {page-component-title} platform, so it's critical to have a firm grasp of this topic.
+
+IMPORTANT: Whenever something in {page-component-title} appears to work by magic, it's probably events working behind the curtain.
```

Comment:
We can remove the important, and just have this as a second sentence.

Reply:
```suggestion
Whenever something in {page-component-title} appears to work by magic, it's probably events working behind the curtain.
```